noir-projects/aztec-nr/aztec/src/context/avm_context.nr

```diff
@@ -200,9 +197,12 @@ impl Empty for AvmContext {
 
 // Helper functions
 fn gas_for_call(user_gas: GasOpts) -> [Field; 2] {
+    // This will get capped by the actual gas left.
+    // TODO: replace with max possible field.
+    let MAX_POSSIBLE_GAS: Field = 1_000_000_000_000_000_000_000;
     [
-        user_gas.l2_gas.unwrap_or_else(|| l2_gas_left() - NESTED_CALL_L2_GAS_BUFFER),
-        user_gas.da_gas.unwrap_or_else(|| da_gas_left())
+        user_gas.l2_gas.unwrap_or_else(|| MAX_POSSIBLE_GAS),
+        user_gas.da_gas.unwrap_or_else(|| MAX_POSSIBLE_GAS)
```

Comment:
Can use max possible gas here because inside of the `CALL` instruction's execute() it caps at gas left?